data.sql

```diff
@@ -47,3 +47,42 @@ SET owner_id = CASE
                 WHEN name IN ('Angemon', 'Boarmon') THEN 5
                 ELSE NULL
               END;
+
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES
+    ('William Tatcher', 45, '2000-04-23'),
+    ('Maisy Smith', 26, '2019-01-17'),
+    ('Stephanie Mendez', 64, '1981-05-04'),
+    ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES
+    (1, 1),
+    (3, 1),
+    (3, 2),
+    (4, 2);
```

Comment:
- [ ] [OPTIONAL] This method of inserting the IDs is okay and it works but it can lead to wrong data input if there is a mistake. Also, what about the situation in which you cannot really retrieve the Vet_id and species_id? The method below will dynamically retrieve and insert the correct data. it appears as more code but it is solid and more reliable.

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```